test/e2e/framework.go

```diff
@@ -127,6 +127,17 @@ type PodIPs struct {
 	ipStrings []string
 }
 
+func (p PodIPs) String() string {
```

Comment:
Is it called some where?

Reply:
It should be called here.
https://github.com/vmware-tanzu/antrea/blob/master/test/e2e/connectivity_test.go#L40

Reply:
Ok